### PR TITLE
impruved #7

### PR DESCRIPTION
**Pull request by 6543**
* resolve conflicts from #7
* revers some unrelated changes
* beautify css 

credits to: @FanyangMengDev
close #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+
+![preview 6](preview/markdowneditor.png "Preview")
+
+![preview 7](preview/profilepage.png "Preview")
+
+![preview 8](preview/settingpage.png "Preview")
+
+
+
+## For Gitea Version ***1.11.0-rc2***
```
Comment (6543) [inline README.md]: unrelated!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,25 @@
-# Gitea Matrix Template- Turn your gitea arc-green template into the matrix
+# Gitea Matrix Template - Make your Gitea cooler
```
Comment (6543) [inline README.md]: unrelated

**Review by 6543 — COMMENTED** [review]
.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,23 @@
 # Gitea Matrix Template- Turn your gitea arc-green template into the matrix
+
 ## Previews:
 
-![preview 1](matrix-preview.png "Preview")
+![preview 1](preview/homepage.png "Preview")
+
+![preview 2](preview/explore.png "Preview")
+
+![preview 3](preview/login.png "Preview")
```
Comment (6543) [inline README.md:9]: do we need ths new previews?
@TylerByte666 what do you think?

File: README.md [hunk 4]
```diff
@@ -21,10 +35,7 @@ DEFAULT_THEME = matrix
 ### Welcome to the matrix!
 
 ### Special Notes:
- - A user can still change the theme settings. 
+ - A user can still change the theme settings.
 
 ### Special Thanks to 6543!
 https://github.com/6543
```
Comment (6543) [inline README.md:41]:
```suggestion
https://github.com/6543
https://github.com/FanyangMengDev
```

File: README.md [hunk 5]
```diff
@@ -21,10 +35,7 @@ DEFAULT_THEME = matrix
 ### Welcome to the matrix!
 
 ### Special Notes:
- - A user can still change the theme settings. 
+ - A user can still change the theme settings.
 
 ### Special Thanks to 6543!
```
Comment (6543) [inline README.md]:
```suggestion
### Special Thanks to 6543 & FanyangMengDev
```

**Review by 6543 — COMMENTED** [review]
add FanyangMengDev to

File: README.md (hunk 3 above)
Comment (TylerByte666) [reply README.md:9]: I think just one preview, we need to show the people what it looks like

Comment (6543) [reply README.md:9]: Okbyouvalready merged ... so I now the new previes are in ... I think the best way now is you change those previws youreself in masterbranch
